resources/windows/wix/condition.wxs.erb

```diff
@@ -18,11 +18,10 @@
         <![CDATA[Installed OR (VersionNT >= 600)]]>
       </Condition>
 
-      <!--
       <Condition Message="32 bit <%= settings[:product_name] %> is no longer supported on 64 bit Windows Operating Systems.">
-          <![CDATA[Installed OR VersionNT64]]>
+          <![CDATA[Installed OR Not VersionNT64]]>
```

Comment:
This condition is kind of confusing... and I think it might have been correct as-is with `Installed OR VersionNT64` -- or even more simply as `VersionNT64`.

According to MSDN, about `VersionNT64`:


> VersionNT64 property
> The installer sets the VersionNT64 property to the version number for the operating system only if the system is running on a 64-bit computer. The property is undefined if the operating system is not 64-bit.

Per: https://msdn.microsoft.com/en-us/library/windows/desktop/aa372497%28v=vs.85%29.aspx

Since this condition is already in a conditionally included block for only the 32-bit installer, if `VersionNT64` is set, then it means we're on 64-bit Windows and should fail to install.

As for the `Installed` property, I don't understand why it's needed here (I know that it was already there but might be worth cleaning up) - see https://msdn.microsoft.com/en-us/library/windows/desktop/aa369297%28v=vs.85%29.aspx